tools/pipelines/templates/upload-dev-manifest.yml

```diff
@@ -13,7 +13,8 @@ jobs:
     - name: version
       value: $[stageDependencies.build.build.outputs['SetVersion.version']]
     steps:
-    - template: include-install-build-tools.yml
+    - template: /tools/pipelines/templates/include-use-node-version.yml@self
+    - template: /tools/pipelines/templates/include-install-build-tools.yml@self
```

Comment:
This was the missing template include, and I standardized the path to include-install-build-tools.yml, to match how we use them everywhere else.